staging/velero/templates/deployment.yaml

```diff
@@ -120,7 +120,7 @@ spec:
       initContainers:
         {{- with .Values.minioBackend }}
         - name: initialize-velero
-          image: mesosphere/kubeaddons-addon-initializer:v0.0.4
+          image: mesosphere/kubeaddons-addon-initializer:v0.0.5
```

Comment:
what did it change at this side ?

Reply:
https://github.com/mesosphere/kubeaddons-extrasteps/pull/10 -- basically some fields in the new minio-opeator are gone now. and it updated names of some resources.

Reply:
ack